### DISCUSSION TITLE
More folders in screenshots

### DISCUSSION THREAD

**Discussion by nmschuler** [runelite/runelite#16298] [Ideas]
When a screen shot is taken for a boss kill, collection log, etc, they are put into their respective folders. However, with specifically boss kill screen shots, they add up quickly. My Boss Kills folder is about 9000 entries. Whenever I open the folder to look at the last time I did a boss, or look for something, it takes a while for windows explorer to organize the files. 

Can we have sub folders for each boss under the main "Boss Kills" folder? So a folder would be created for "Abyssal Sire", then every Abyssal Sire boss kill screen shot would go in there, rather 1 large dump of all boss kills.

**Comment by MoonWolf125** [comment]
At the risk of Necroing an old discussion...

This is definitely an issue for those of us who boss in high amounts, even with top of the line hardware. The default folder sort is by _Name Ascending_ generally many of us are opting to enforce a different sort like _Date Descending_ so we can see the most recent screenshots. For example, before I discovered that the Discord Rare Drop Notifier plugin was abandoned my friends and I were posting our Yama loot/collection logs manually. Instead of waiting for the file system to read a few thousand images to enforce the sort it would make a lot more sense to have a selection option within the Screenshots plugin to further subdivide the images into folders automatically. Doubly so because it would be far faster for us to simply click on the new Boss' folder.

In fact, there's been multiple requests for a similar update in the past dating as far back as 2018:
https://github.com/runelite/runelite/issues/2247, https://github.com/runelite/runelite/issues/13553, https://github.com/runelite/runelite/issues/14695

At this point this isn't a random problem solved by using built in file system search/sort/group features. The original issue that led to the implementation of the Boss Kills folder in 2019 (https://github.com/runelite/runelite/issues/8711) was created because of this same exact issue - too many screenshots bogging down File Explorers and that was resolved by creating sub-directories. I implore the development team to accept a sub-directory solution for this continued issue.
